doc/en/components/link-button.md

```diff
@@ -0,0 +1,208 @@
+# $Platform$ Link Button Overview
+
+The $Platform$ Link Button provides different variants and styles, which are easily configurable to match the appearance and functionality, that we want to achieve.
+
+## $Platform$ Link Button Example
+
+<code-view style="height: 100px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/inputs/link-button-overview"
+           alt="$Platform$ Button Example"
+           github-src="inputs/link-button/overview">
+</code-view>
+
+## Usage
+
+To get started with the Link Button Web Component we need to import the `IgcLinkButtonComponent` in our typescript file:
+
+```ts
+import { IgcLinkButtonComponent } from 'igniteui-webcomponents'; 
+```
+
+After we import the link button component, we are ready to start using it, so let's add our first link button.
+
+```html
+<igc-link-button
+    variant="contained"
+    href="https://www.infragistics.com/products/ignite-ui-web-components"
+    rel="external"
+    target="_blank">
+    <span slot="prefix">+</span>
+     Click Me
+    <span slot="suffix">-</span>
+</igc-link-button>
+```
+
+With `prefix` and `suffix` slots we can add different content before and after the main content of the button.
+
+The internal structure of the `igc-link-button` is an [`<a>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) type element, which is why we can
+set the `href`, `rel`, `target` and `download` attributes of the link button.
+
+## Link Button Variants
+
+### Flat Button
+
+Use `flat` variant to add a simple flat button in your component template. Note that if you do not set variant, by default it will be set to flat.
+
+```html
+<igc-link-button
+    variant="flat"
+    href="https://www.infragistics.com/webcomponentssite/components/general-getting-started.html">
+    Flat
+</igc-link-button>
+```
+
+<div class="sample-container loading" style="height: 70px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/link-button-flat">
+</iframe></div>
```

Comment:
Shouldn't we use `<code-view>` instead?

Reply:
I follow this approach from Angular button topic.